example/starwars/starwars.go

```diff
@@ -332,7 +328,13 @@ func (r *Resolver) Character(args struct{ ID graphql.ID }) *characterResolver {
 }
 
 func (r *Resolver) Human(args struct{ ID graphql.ID }) *humanResolver {
+	startTime := time.Now()
+	for _, h := range humans {
+		humanData[h.ID] = h
+	}
 	if h := humanData[args.ID]; h != nil {
+		endTime := time.Now()
+		log.Println("duration:", endTime.Sub(startTime).Seconds())
```

Comment:
Last note, just a word of caution: the `log` loggers will serialize writes to whatever Writer they have been initialized with. Depending on what rate the resolvers are being executed, this may cause your application to begin to deadlock on this log Writer.